shared/settings/account/index.tsx

```diff
@@ -2,6 +2,7 @@ import * as React from 'react'
 import * as Kb from '../../common-adapters'
 import * as Styles from '../../styles'
 import EmailPhoneRow from './email-phone-row'
+import * as I from 'immutable'
 
 type Props = {
   contactKeys: Array<string>
```

Comment:
did you intend to make this `I.List`?

Reply:
I'm not sure. I initially made it that, but then I got some TS errors. Will this render-thrash?

Reply:
It shouldn't thrash, you're passing in a `List` though. L25 isn't erroring because `props` is untyped in the subcomponents, we should add that.